### ISSUE TITLE
NVDA is unable to interact with Skype Electron at all

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#11954]
### Steps to reproduce:

1. Launch skype electron and attempt to do anything. E.g, tabbing around, pressing hotkeys etc.

### Actual behavior:

Nothing is read at all, nothing is visible with object navigation except for the system menu and the title bar.

The workaround of pressing alt and then pressing escape which used to work to somehow correct focus no longer works now.

### Expected behavior:

Skype reads properly with NVDA

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-21524,7b92e442

#### Windows version:
Windows 10 2004

#### Name and version of other software in use when reproducing the issue:
Skype 8 (Electron)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, previous snapshot versions

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by seanbudd** [comment]
Is this still an issue with the latest version of Skype and NVDA 2022.1rc2?

**Comment by seanbudd** [comment]
@k-kolev1985 - can you confirm this is no longer an issue as well?

**Comment by k-kolev1985** [comment]
Yes, I can confirm this is no longer an issue.

**Comment by seanbudd** [comment]
Thanks
